README.md

```diff
@@ -286,6 +286,20 @@ Pretty.prettyPrint(value, option: Pretty.Option(prefix: nil, indentSize: 2))
 // )
 ```
 
+### colorized
+
+Output strings can be colored.
```

Comment:
Please write more detail description.

For example:

- Colorlized are works by ANSI color.
- Therefore it works only terminal that supported ANSI color e.g. AppCode, Terminal.app (This is ** not include Xcode**).
- Related to [terminal](https://github.com/YusukeHosonuma/SwiftPrettyPrint#terminal) section.

Reply:
Add detail description